admin_guide/limits.adoc

```diff
@@ -103,38 +108,46 @@ Per container, the following must hold true if specified:
 |Constraint |Behavior
 
 |`*Min*`
-|Min[resource] <= container.resources.requests[resource] (required) <= container/resources.limits[resource] (optional)
+|`Min[resource]` less than or equal to `container.resources.requests[resource]`
+(required) less than or equal to `container/resources.limits[resource]`
```

Comment:
Is the `/` a typo?  That is, i think the (full) field name would be `container.resources.limits`.